src/Stepper/index.js

```diff
@@ -4,79 +4,71 @@ import * as React from "react";
 import StepperStateless from "./StepperStateless";
 import validateIncrement from "../utils/validateIncrement";
 import validateDecrement from "../utils/validateDecrement";
+import useStateWithCallback from "../hooks/useStateWithCallback";
 
-import type { Props, State } from "./index";
+import type { Props } from "./index";
 
-class Stepper extends React.PureComponent<Props, State> {
-  state = {
-    value: this.props.defaultValue || 0,
-  };
-
-  setValueAndInjectCallback = (value: number) => {
-    const { onChange } = this.props;
+const Stepper = ({ onChange, defaultValue, ...props }: Props) => {
```

Comment:
I would unify the approach when comes to default values. e.g. `defaultValue = 0` and don't use it on line 12.